### PR TITLE
Add initial unit tests

### PR DESCRIPTION
**Pull request by matejkrajcovic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+notifications:
```
Comment (misogl) [inline .travis.yml]: We don't use Travis anymore.

File: package.json [hunk 2]
```diff
@@ -3,11 +3,12 @@
   "version": "1.0.3",
   "description": "Lunch bot for Vacuumlabs lunch orders in Slack",
   "scripts": {
-    "start": "npm run build && node server.js",
+    "start": "yarn run build && node server.js",
     "dev": "babel-watch server.js",
-    "test": "eslint src/**",
+    "test": "yarn run build && yarn run eslint && yarn run mocha",
```
Comment (misogl) [inline package.json]: You should run build after `eslint`.

File: config.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // load .env variables into process.env
-require('dotenv').config();
+require('dotenv').config({path: process.env.TEST ? './.env.test' : './.env'});
```
Comment (misogl) [inline config.js]: `process.env.CIRCLECI` is already defined.

**Review by misogl — CHANGES_REQUESTED** [review]
I don't find the tests very useful. You test only basic orders and don't test invalid orders (just non-orders).
Also: why did you amend someone's else commit that was 1 year old? I would expect a rebase and a separate fix commit.
BTW: the original rename of `isChannelPublic` to `isChannelMain` was a good change.

**Comment by matejkrajcovic** [comment]
@misogl These are only basic unit tests to prepare infrastructure. In #85 I add new regexp tests.

File: package.json [hunk 4]
```diff
@@ -3,11 +3,12 @@
...
   "scripts": {
-    "start": "npm run build && node server.js",
+    "start": "yarn run build && node server.js",
     "dev": "babel-watch server.js",
-    "test": "eslint src/**",
+    "test": "yarn run eslint && yarn run build && yarn run mocha",
     "build": "babel src/ -d build/ -s",
-    "eslint": "eslint src/**"
+    "eslint": "eslint src/** test/**",
+    "mocha": "CIRCLECI=1 mocha --require babel-core/register"
```
Comment (misogl) [inline package.json:11]: `CIRCLECI` is already defined.

**Review by misogl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**